docs/Installation/Installation_Guide.rst

```diff
@@ -100,19 +98,18 @@ Installing Zenko
 ################
 
 Use a Fresh Cluster
-===================
+*******************
 
 If you are installing Zenko on a fresh cluster, the following instructions
 are sufficient. If, however, you are reinstalling Zenko on an existing cluster
 (while assessing or testing Zenko, for example), it is a best practice to use
-a fresh cluster.
+a fresh cluster. If you cannot reinstall to a fresh cluster (for a dep
```

Comment:
? Unfinished sentence

Reply:
Also, this paragraph as a whole is quite redundant - the 'however' part still goes back to repeating that the user should use a fresh cluster, rather than directing the user to a new set of instructions to reinstall on a previously used  cluster.

Reply:
Use a Fresh Cluster
*******************

It is a best practice to install Zenko on a fresh cluster.

From a bare cluster, be prepared to install basic utilities that you will need
to deploy. If you intend to deploy using Git, for example, you'll need to
install it first. If you aren't comfortable with vi-like text editors (vim,
e.g.) install your preferred editor.

Reply:
That's it.